_about/notes/0.5.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Istio 0.5
+
+order: 96
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Alpha2 Traffic Rules**. Add support for an updated model of routing and destination rules. TODO: Doc link?
+
+- **Control Plane Deployments**. `istioctl` now has a deployment command to produce istio control plane deployments. TODO: Doc link?
```

Comment:
cc @ZackButcher

Reply:
We need to highlight this and actually make some noise. This essentially handles the piecemeal installation.

Reply:
Can someone supply better phrasing ?

Reply:
I'm not sure it's ready for prime time yet, the set of options it allows isn't as extensible as we'd like, and I don't think we have stable testing for it yet (e.g. we'd like to use it to deploy all of our e2e tests during presubmit, etc). cc @costinm thoughts here, you've been working in this space too.

Reply:
Yeah, but its a step in the right direction. May be prefix it with something like a "preview" release or alpha version of Istio custom installer for folks seeking incremental adoption of Istio.

Reply:
That's fair.

Reply:
Zack, can you give me the phrasing to insert here?

Thanks.